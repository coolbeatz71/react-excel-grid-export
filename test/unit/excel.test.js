import React from "react";
import { shallow } from "enzyme";

import GridColumn from "../../src/GridPlugin/elements/GridColumn";

describe("React Data Export unit test", () => {
  test("should render GridColumn", () => {
    const wrapper = shallow(<GridColumn label={"test"} value={"test"} />);

    expect(wrapper.html()).toMatchSnapshot();
  });
});
