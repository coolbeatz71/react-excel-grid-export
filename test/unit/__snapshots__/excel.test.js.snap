// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`React Data Export unit test should render GridColumn 1`] = `null`;
